schema-engine/core/src/commands/create_migration.rs

```diff
@@ -1,8 +1,15 @@
 use crate::{json_rpc::types::*, CoreError, CoreResult, SchemaContainerExt};
+use crosstarget_utils::time::format_utc_now;
 use schema_connector::{migrations_directory::*, DiffTarget, SchemaConnector};
-use std::path::Path;
 use user_facing_errors::schema_engine::MigrationNameTooLong;
 
+/// Create a directory name for a new migration.
+pub fn generate_migration_directory_name(migration_name: &str) -> String {
+    let timestamp = format_utc_now("%Y%m%d%H%M%S");
```

Comment:
The timestamp generation still needs to happen in Rust, because of how `sql-migration-tests` rely on it.